### DISCUSSION TITLE
Is it possible to display raw signals in Insight?

### DISCUSSION THREAD

**Discussion by dyang** [TelemetryDeck/Viewer#295] [General]
I have set up a few time-series Insights with filters applied. Currently, it appears that I can only see the signal or user counts on the Y-axis, e.g., I can view the total number of signals a certain user sent per day. I am wondering if it's possible to display the raw signals? I'd like to study these signals to better understand how this specific user interacts with my app. Thanks!

**Comment by winsmith** [comment]
Unfortunately, an insight cannot easily display raw signals. You can look at raw signals using the "Recent Signals" tab, however. 

<img width="1342" alt="Screenshot 2023-10-20 at 13 54 32" src="https://github.com/TelemetryDeck/Issues/assets/954120/d11f1ae1-f80e-4041-9125-9f109d84c498">
